nengo_loihi/builder.py

```diff
@@ -731,6 +524,7 @@ def signal_probe(model, key, probe):
     model.objs[probe]['out'] = cx_probe
 
 
+# TODO: why q, s, v, u ?
```

Comment:
These correspond to the variable names used in `CxSimulator`, which in turn were based off those in the mathematical overview of the chip logic provided by Intel.